HISTORY.rst

```diff
@@ -18,6 +18,12 @@ History
 * Update check method for calibration
 * Fix bug in MapieClassifier with different number of labels in calibration dataset.
 
+
+0.6.5(2023-07-11)
+------------------
+
+* Add LTT for multilabel-classification
```

Comment:
UPDATE: must be indicated on line 8 without specifying the version and date (this is managed when a release is made).